containers/podman-desktop
# chore: add typecheck verification

### What does this PR do?
Vite is not checking the typescript so we may have typescript errors over time and we should check that on PR checks
https://vitejs.dev/guide/features.html#typescript

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

N/A

### How to test this PR?

PR check should be green ✅ 
App should work as before

Change-Id: I2f19b8797ad861e8594faef2e366a200382a4923
Signed-off-by: Florent Benoit <fbenoit@redhat.com>
